bugbug/utils.py

```diff
@@ -245,14 +245,32 @@ def zstd_compress(path: str) -> None:
     if not os.path.exists(path):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), path)
 
-    subprocess.run(["zstdmt", "-f", path], check=True)
+    try:
+        subprocess.run(["zstdmt", "-f", path], check=True)
+    except FileNotFoundError:
+        logger.warning(
+            "zstdmt binary not present. Falling back to zstandard API, which could be slower."
```

Comment:
Many things could go wrong. It could be better to print out the actual error instead of making assumptions:
```suggestion
    except FileNotFoundError as err:
        logger.warning(
            "%s. Falling back to zstandard API, which could be slower.", err
```